gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/conn/GenericLanguageConnectionContext.java

```diff
@@ -487,12 +489,16 @@ protected SchemaDescriptor initDefaultSchemaDescriptor()
         if (cachedInitialDefaultSchemaDescr == null) {
             DataDictionary dd = getDataDictionary();
             String authorizationId = getAuthorizationId();
+			String defaultSchema = authorizationId.replace('-', '_');
             SchemaDescriptor sd =
                 dd.getSchemaDescriptor(
-                    authorizationId, getTransactionCompile(), false);
+                    defaultSchema, getTransactionCompile(), false);
+			LogWriter logger = GemFireCacheImpl.getExisting().getLogger();
+			logger.warning("Setting the default schema with authid conditioned as= " + authorizationId );
```

Comment:
You want to print authorizationId or defaultSchema?

Reply:
actually nothing . removing it.